src/query.js

```diff
@@ -14,6 +14,17 @@ export function reduce(iterable, fn, initial) {
   return query(iterable).reduce(fn, initial);
 }
 
+export class Entry {
+  constructor(key, value) {
+    this.key = key;
+    this.value = value;
+  }
+
+  [Symbol.iterator]() {
+    return [this.value, this.key][Symbol.iterator]();
```

Comment:
Sometimes you come across a line of code that is just pleasing to look at. This is one of those lines.

Reply:
🤗